tests/snmpsim/ird.snmprec

```diff
@@ -1,143 +1,145 @@
- 1.3.6.1.4.1.1070.3.1.1.104.9.4.3.0|4x|00000000000000000000000000000000
- 1.3.6.1.4.1.1070.3.1.1.104.9.4.4.0|4x|0000000000000000000000000000
- 1.3.6.1.4.1.1070.3.1.1.104.10.1.0|64x|0a03008f
- 1.3.6.1.4.1.1070.3.1.1.104.10.2.0|2|2000
- 1.3.6.1.4.1.1070.3.1.1.104.10.3.0|64x|e1010101
- 1.3.6.1.4.1.1070.3.1.1.104.10.4.0|2|7
- 1.3.6.1.4.1.1070.3.1.1.104.10.5.0|2|10
- 1.3.6.1.4.1.1070.3.1.1.104.10.6.0|64x|ffffff00
- 1.3.6.1.4.1.1070.3.1.1.104.10.7.0|64x|0a030001
- 1.3.6.1.4.1.1070.3.1.1.104.10.8.0|4x|000e26ffba39
- 1.3.6.1.4.1.1070.3.1.1.104.10.9.0|2|3000
- 1.3.6.1.4.1.1070.3.1.1.104.10.10.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.10.11.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.10.12.0|2|3
- 1.3.6.1.4.1.1070.3.1.1.104.10.13.0|2|1
- 1.3.6.1.4.1.1070.3.1.1.104.10.14.0|2|6
- 1.3.6.1.4.1.1070.3.1.1.104.10.15.0|2|6
- 1.3.6.1.4.1.1070.3.1.1.104.10.16.0|2|5
- 1.3.6.1.4.1.1070.3.1.1.104.10.17.0|4x|00ef014e08d2040000d50700000000
- 1.3.6.1.4.1.1070.3.1.1.104.10.18.0|4x|66663a66663a66663a66663a66663a6666
- 1.3.6.1.4.1.1070.3.1.1.104.10.19.0|4x|34120000463a46463a46463a46463a464600000000000000
- 1.3.6.1.4.1.1070.3.1.1.104.10.20.0|4x|56220000463a00003a460000464600004600000000000000
- 1.3.6.1.4.1.1070.3.1.1.104.10.21.0|4x|30303a30303a31323a33343a32323a3536
- 1.3.6.1.4.1.1070.3.1.1.104.10.24.0|4|
- 1.3.6.1.4.1.1070.3.1.1.104.10.25.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.11.1.0|2|1
- 1.3.6.1.4.1.1070.3.1.1.104.11.2.0|2|14
- 1.3.6.1.4.1.1070.3.1.1.104.11.3.0|4x|5e766576
- 1.3.6.1.4.1.1070.3.1.1.104.11.4.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.11.5.0|4|
- 1.3.6.1.4.1.1070.3.1.1.104.11.6.0|2|1
- 1.3.6.1.4.1.1070.3.1.1.104.11.7.0|4|
- 1.3.6.1.4.1.1070.3.1.1.104.11.8.0|4|
- 1.3.6.1.4.1.1070.3.1.1.104.11.9.0|4x|0243727970746f776f726b73
- 1.3.6.1.4.1.1070.3.1.1.104.11.10.0|4x|01
- 1.3.6.1.4.1.1070.3.1.1.104.11.11.0|4x|01000000
- 1.3.6.1.4.1.1070.3.1.1.104.13.1.0|64x|0a03008f
- 1.3.6.1.4.1.1070.3.1.1.104.13.2.0|64x|ffffff00
- 1.3.6.1.4.1.1070.3.1.1.104.13.3.0|64x|0a030001
- 1.3.6.1.4.1.1070.3.1.1.104.13.4.0|4x|000e26ffba39
- 1.3.6.1.4.1.1070.3.1.1.104.13.5.0|64x|00000000
- 1.3.6.1.4.1.1070.3.1.1.104.13.6.0|2|3000
- 1.3.6.1.4.1.1070.3.1.1.104.13.7.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.13.8.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.13.9.0|2|38000
- 1.3.6.1.4.1.1070.3.1.1.104.13.10.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.14.1.0|2|10600
- 1.3.6.1.4.1.1070.3.1.1.104.14.2.0|2|11804
- 1.3.6.1.4.1.1070.3.1.1.104.14.3.0|2|28000
- 1.3.6.1.4.1.1070.3.1.1.104.14.4.0|2|1
- 1.3.6.1.4.1.1070.3.1.1.104.14.5.0|2|1
- 1.3.6.1.4.1.1070.3.1.1.104.14.6.0|2|1
- 1.3.6.1.4.1.1070.3.1.1.104.14.7.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.14.8.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.14.9.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.14.10.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.15.1.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.16.1.0|2|0
- 1.3.6.1.4.1.1070.3.1.1.104.16.2.0|4|
- 1.3.6.1.4.1.1070.3.1.1.104.16.3.0|2|1
- 1.3.6.1.4.1.1070.3.1.1.104.16.4.0|2|50000
- 1.3.6.1.4.1.1070.3.1.1.104.17.1.0|4|
- 1.3.6.1.4.1.1070.3.1.1.104.17.2.0|4|
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2021.250.10
```

Comment:
This should probably be a different test file, it doesn't seem to have all the sensors the other file has.